pages/index.tsx

```diff
@@ -44,7 +76,19 @@ export default function Home() {
         setVisibleDistrictIndices([...visibleDistrictIndices, districtIndex]);
         setError('');
       } else {
-        setError(`No district named ${userInput}`);
+        const possibleDistrictIndex = DistrictData.findIndex((d) =>
+          stringSimilarity.compareTwoStrings(
+            d.name.toLowerCase(),
+            userInput.toLowerCase()
+          ) > 0.6
+            ? true
+            : false
+        );
```

Comment:
you could just return the result of the condition, no need for the ternary operator